static/js/recent_topics_ui.js

```diff
@@ -116,6 +124,35 @@ function is_table_focused() {
     return $current_focus_elem === "table";
 }
 
+function get_row_type(row) {
+    // Return "private" or "stream"
+    // list_widget.clean_redraw() calls get_current_list before
+    // the widget is returned and thus initialized.
+    // So we use CSS method for finding row type until topics_widget gets initialized.
+    if (!topics_widget) {
+        const $topic_rows = $("#recent_topics_table table tbody tr");
+        const $topic_row = $topic_rows.eq(row);
```

Comment:
I don't understand the `.find` here; how is this different from just `const $topic_row = row`?